Yoast/wordpress-seo
# Stories/9327 hide marker buttons when in gutenberg

## Summary

This PR can be summarized in the following changelog entry:

* Hides the marker buttons when in Gutenberg.

## Relevant technical choices:

* Adds a way to check if Gutenberg Post is available.

## Test instructions

This PR can be tested by following these steps:

* Go to edit a post.
* Switch between the classic editor and Gutenberg and check if the marker buttons are there in the classic editor and not there in Gutenberg.

## Quality assurance


Fixes #9327
